Main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+  "geoffrey/commands"
+
+  "os"

```

Comment:
look into [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) for info on how go likes its stuff imported.

The rule is top group is stdlib so stuff like `os`. Second group is third party. Last group is stuff thats in this project.